src/e2e/java/teammates/e2e/pageobjects/AdminNotificationsPage.java

```diff
@@ -0,0 +1,240 @@
+            WebElement notificationRow = notificationsTable.findElement(By.id(notification.getNotificationId()));
+            verifyTableRowValues(notificationRow, getNotificationTableDisplayDetails(notification));
+        }
+    }
+
+    public void verifyNotificationAttributes(NotificationAttributes expected, NotificationAttributes actual) {
+        // Notification ID is not checked
+        // as the actual notification has a randomly generated ID
+        assertEquals(expected.getStartTime(), actual.getStartTime());
+        assertEquals(expected.getEndTime(), actual.getEndTime());
+        assertEquals(expected.getStyle(), actual.getStyle());
+        assertEquals(expected.getTargetUser(), actual.getTargetUser());
+        assertEquals(expected.getTitle(), actual.getTitle());
+        assertEquals(expected.getMessage(), actual.getMessage());
+    }
+
+    public void addNotification(NotificationAttributes notification) {
+        clickAddNotificationButton();
+        waitForElementPresence(By.id("btn-create-notification"));
+
+        fillNotificationForm(notification);
+
+        clickCreateNotificationButton();
+        waitForPageToLoad(true);
+    }
+
+    public void editNotification(NotificationAttributes notification) {
+        WebElement notificationRow = notificationsTable.findElement(By.id(notification.getNotificationId()));
+        WebElement editButton = notificationRow.findElement(By.className("btn-light"));
+        editButton.click();
+        waitForElementPresence(By.id("btn-edit-notification"));
+
+        fillNotificationForm(notification);
+
+        clickEditNotificationButton();
+        waitForPageToLoad(true);
+    }
+
+    public void deleteNotification(NotificationAttributes notification) {
+        WebElement notificationRow = notificationsTable.findElement(By.id(notification.getNotificationId()));
+        WebElement deleteButton = notificationRow.findElement(By.className("btn-danger"));
+
+        deleteButton.click();
+        waitForConfirmationModalAndClickOk();
+        waitForPageToLoad(true);
+    }
+
+    public void fillNotificationForm(NotificationAttributes notification) {
+        selectDropdownOptionByText(notificationTargetUserDropdown, getTargetUserText(notification.getTargetUser()));
+        selectDropdownOptionByText(notificationStyleDropdown, getNotificationStyle(notification.getStyle()));
+        fillTextBox(notificationTitleTextBox, notification.getTitle());
+        setMessage(notification.getMessage());
+        setNotificationStartDateTime(notification.getStartTime());
+        setNotificationEndDateTime(notification.getEndTime());
+    }
+
+    public String getNewestNotificationId() {
+        // Sorts notifications by descending creation time
+        WebElement creationTimeHeader = notificationsTable.findElements(By.tagName("th")).get(5);
+        if (creationTimeHeader.findElements(By.className("fa-sort-down")).size() == 0) {
+            click(creationTimeHeader);
+        }
```

Comment:
The sorting should be done outside `getNewestNotificationId` in another method e.g. `sortByCreationTime`, by its method name the responsibility of `getNewestNotificationId` is just to get the top row notification id